bamboo-storage/src/main/java/com/pushtorefresh/android/bamboostorage/operation/get/PreparedGetAsCursor.java

```diff
@@ -29,4 +30,24 @@ public PreparedGetWithResultAsCursor(@NonNull BambooStorage bambooStorage, @NonN
             }
         });
     }
+
+    public static class Builder {
+
+        @NonNull private final BambooStorage bambooStorage;
+
+        private Query query;
+
+        public Builder(@NonNull BambooStorage bambooStorage) {
+            this.bambooStorage = bambooStorage;
+        }
+
+        @NonNull public Builder query(@NonNull Query query) {
+            this.query = query;
+            return this;
+        }
+
+        @NonNull public PreparedOperation<Cursor> prepare() {
+            return new PreparedGetAsCursor(bambooStorage, query);
```

Comment:
Yep, it's an API draft implementation, I've created an issue to write checks for all Builder classes #43